exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/DrillSqlResetOption.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.planner.sql.parser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlLiteral;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlSetOption;
+import org.apache.calcite.sql.SqlSpecialOperator;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.util.ImmutableNullableList;
+
+/**
+ * Sql parse tree node to represent statement:
+ * {@code RESET { <NAME> | ALL } }
+ * <p><p>
+ * Statement handled in: {@link org.apache.drill.exec.planner.sql.handlers.SetAndResetOptionHandler}
+ */
+public final class DrillSqlResetOption extends SqlSetOption {
+  private final static SqlKind SQL_KIND = SqlKind.SET_OPTION;
+
+  public final static SqlSpecialOperator OPERATOR = new SqlSpecialOperator("RESET_OPTION", SQL_KIND) {
```

Comment:
Please reorder modifiers according to JLS here and in other places.
```suggestion
  public static final SqlSpecialOperator OPERATOR = new SqlSpecialOperator("RESET_OPTION", SQL_KIND) {
```